docs/fast_data/single_view_trigger_generator.md

```diff
@@ -4,16 +4,12 @@ title: Single View Trigger Generator
 sidebar_label: Single View Trigger Generator
 ---
 
-:::caution
-This Plugin is a BETA Plugin and, as such, is currently under active development. Pay attention using it.
-:::
-
-The Single View Trigger Generator is a new component of the Fast Data architecture. Its introduction allows splitting the import phase
-from [strategies](/fast_data/the_basics.md#strategies) execution, which are currently handled both by the [Real-Time Updater](/fast_data/realtime_updater.md).
+The Single View Trigger Generator (SVTG) is another component of the Fast Data architecture that can be employed in the event-driver architectures.
+Its introduction allows splitting the import phase from [strategies](/fast_data/the_basics.md#strategies) execution, which are currently handled both by the [Real-Time Updater](/fast_data/realtime_updater.md).
```

Comment:
@danibix95 credo manchi la fine della frase qui

Reply:
Actually, the sentence should end like this. The idea is to say that currently RTU is handling both import and strategy generation, while starting with event-driven architecture, the latter feature is responsibility of SVTG.

Let me know whether the description is not clear enough or misleading.

Reply:
Allora possiamo riscriverla così: 

The Single View Trigger Generator (SVTG) is another component of the Fast Data architecture that can be employed in the event-driven architectures.
Its introduction allows splitting the data import phase (ingesting data from topics) from [strategies](/fast_data/the_basics.md#strategies) execution, which in the Standard architecture are both handled by the [Real-Time Updater](/fast_data/realtime_updater.md).

Che ne pensi?

Aggiungiamo magari un link alla doc delle architecture? Lo screenshot in cui c'è anche il Projection Storer è ancora da mettere?

Reply:
Updated! The screenshots have already been updated 👍🏻